railties/lib/rails/generators/rails/app/templates/config/environments/production.rb.tt

```diff
@@ -55,6 +55,7 @@ Rails.application.configure do
 
   # Prepend all log lines with the following tags.
   # config.log_tags = [ :subdomain, :request_id ]
+  config.log_tags = [ :request_id ]
```

Comment:
Feels weird to having this line almost match the comment above. Can we find some better methods to add or should we ditch it altogether since we're showing usage now?